frontend/src/component/project/NewProjectCard/ProjectModeBadge/ProjectModeBadge.tsx

```diff
@@ -5,7 +5,7 @@ import { HtmlTooltip } from 'component/common/HtmlTooltip/HtmlTooltip';
 import { Badge } from 'component/common/Badge/Badge';
 
 interface IProjectModeBadgeProps {
-    mode: 'private' | 'protected' | 'public' | string;
+    mode?: 'private' | 'protected' | 'public' | string;
```

Comment:
why do we relax the constraint?

Reply:
Component was already written in a way that if any other string is provided, it will be ignored